spacecmd/src/spacecmd/utils.py

```diff
@@ -244,6 +245,7 @@ def editor(template='', delete=False):
             handle = open(file_name, 'r')
             contents = handle.read()
             handle.close()
+            File = namedtuple("file", ("contents", "file_name"))
```

Comment:
Normally, type declarations are done in the module scope. Is there a specific reason you've opted to have it inside the function?